assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -75,70 +75,73 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
     );
   }, []);
 
-  const saveStory = useCallback(() => {
-    setIsSaving(true);
-    const {
-      title,
-      status,
-      author,
-      date,
-      modified,
-      slug,
-      excerpt,
-      featuredMedia,
-      password,
-      publisherLogo,
-    } = story;
+  const saveStory = useCallback(
+    (props) => {
```

Comment:
What this props contains other than status? If not it's better to have it just as status?

Reply:
Indeed. My thought is more related to flexibility, but not a problem change it to status for me. Thanks @ndev1991

Reply:
Since we are considering objectPick/object spread, which is more standard in the project. I will continue using object here. Thanks again.